src/pages/settings/Wallet/ReportFraudPage.js

```diff
@@ -0,0 +1,103 @@
+import React, {useEffect} from 'react';
+import _ from 'underscore';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import ROUTES from '../../../ROUTES';
+import HeaderWithBackButton from '../../../components/HeaderWithBackButton';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import Navigation from '../../../libs/Navigation/Navigation';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+import useLocalize from '../../../hooks/useLocalize';
+import * as Card from '../../../libs/actions/Card';
+import assignedCardPropTypes from './assignedCardPropTypes';
+import * as CardUtils from '../../../libs/CardUtils';
+import ONYXKEYS from '../../../ONYXKEYS';
+import NotFoundPage from '../../ErrorPage/NotFoundPage';
+import usePrevious from '../../../hooks/usePrevious';
+import FormAlertWithSubmitButton from '../../../components/FormAlertWithSubmitButton';
+import * as ErrorUtils from '../../../libs/ErrorUtils';
+
+const propTypes = {
+    /* Onyx Props */
+    formData: PropTypes.shape({
+        isLoading: PropTypes.bool,
+    }),
+    cardList: PropTypes.objectOf(assignedCardPropTypes),
+    /** The parameters needed to authenticate with a short-lived token are in the URL */
+    route: PropTypes.shape({
+        /** Each parameter passed via the URL */
+        params: PropTypes.shape({
+            /** Domain string */
+            domain: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    cardList: {},
+    formData: {},
+};
+
+function ReportFraudPage({
```

Comment:
Can we rename this and the filename to `ReportVirtualCardFraudPage`?

Reply:
Done!